features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/.eslintrc.js

```diff
@@ -43,17 +44,15 @@ module.exports = {
         'import/extensions': ['off'],
         'import/no-named-as-default': ['off'],
         'import/no-named-as-default-member': ['off'],
-        'no-underscore-dangle': [
-            'error',
-            {
-                allowAfterThis: true,
-            },
-        ],
+        'no-underscore-dangle': 0,
         'no-restricted-syntax': ['off'],
+        'no-restricted-globals': ["off"],
         'no-plusplus': ['off'],
+        "no-param-reassign": 0,
```

Comment:
I think this rule prevents following action, and IMHO it's better to keep this rule enabled? 

function (param) {
param = "My Val";
}

Reply:
Yes.. it prevents to do foo.bar = someothervar.param;